wherehows-web/app/components/dataset-aclaccess.ts

```diff
@@ -25,9 +26,11 @@ const millisecondDays = (day = 1) => day * 24 * 60 * 60 * 1000;
  * at least 1 day from now
  * @type {Date}
  */
-// Making min selectable date to be an hour from the current time allows the user to select the
-// next day on the calendar but normally not the current day unless it's 11PM
-const minSelectableExpirationDate = new Date(Date.now() + millisecondDays(1 / 24));
+const minSelectableExpirationDate = new Date(
```

Comment:
Probably do the same for `maxSelectableExpirationDate`

Reply:
Good point, no sense in keeping that util function around when we have moment